cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -153,6 +156,29 @@ function FilterableListControls( element ) {
     return msg || ERROR_MESSAGES.DEFAULT;
   }
 
+  /**
+   * Highlight invalid text fields by giving them an error class.
+   * @param {Array} fields - A list of form fields.
+   * @returns {Array} An array of invalid fields.
+   */
+  function _highlightInvalidFields( fields ) {
+    INVALID_FIELDS.forEach( field => {
+      field.classList.remove( FIELD_ERROR_CLASS );
+    } );
+
+    INVALID_FIELDS = [];
+
+    fields.invalid.forEach( validation => {
+      const field = validation.field;
+      if ( field.type === 'text' ) {
```

Comment:
> @anselmbradford This may become irrelevant after you launch your new datepicker.

Yeah the datepicker will set the field type to type "date", yet it will fall back to a regular text input on Safari/legacy browsers… so, this will stop working where it is needed. Any ideas? Using a `data-` attribute comes to mind, but that feels brittle/messy.

Reply:
We could make this check for the text and date types, and the error state for a modern date input would just never be applied.